lib/PlacesProvider.js

```diff
@@ -204,6 +285,33 @@ Links.prototype = {
   },
 
   /**
+   * Blocks a URL
+   */
+  blockURL(url) {
+    if (this.blockedURLs.save(url) > 0) {
```

Comment:
Personally I think something like returning a boolean/something like this would be more readable:
```js
const didChange = this.blockedURLs.save(url);
if (didChange) {
   this.emit...
}
```
but yeah, idk